WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -1647,20 +1660,27 @@ nick_the_quarreller = {
 nick_the_devil = {
 	allow = {
 		ruled_years = 10
-		trait = wroth
+		OR = {
+			trait = wroth
+			trait = cannibal_trait
+			has_character_modifier = cannibal
+		}
```

Comment:
A cannibal would not be likely to remembered as a Devil, that is for the other traits. For cannibals a separate nickname can be added.